### PR TITLE
add a new page to show how to setup dashboard analytics

### PR DESCRIPTION
**Pull request by sedkis**
Will walk through what the 3 different pumps that tie to Dashboard due.

### CODE REVIEW

**Comment by sedkis** [comment]
this is for :
https://support.tyk.io/a/tickets/4143

**Review by marksou — APPROVED** [review]
Couple of minor edits.

**Review by sedkis — COMMENTED** [review]
thanks!

File: tyk-docs/content/configure/tyk-pump-dashboard-config.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    }
+}
+```
+
+### mongo-pump-aggregate
+This pump stores data in a collection called `z_tyk_analyticz_aggregate_{ORG ID}`.  
+
+There are minimal number of documents that get stored, so you don't need to worry about capping this. The documents contain aggregate info across an individual API, such as total requests, errors, and more.
+
+This pump supplies the data for the following sub categories `API Usage Data`:
```
Comment (komalsukhani) [inline tyk-docs/content/configure/tyk-pump-dashboard-config.md:45]: Worth mentioning, On Dashboard `enable_aggregate_lookups`, should be set to true if mongo aggregate pump is added.

File: tyk-docs/content/configure/tyk-pump-dashboard-config.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+pump config
+```{.json}
+{
+  ...
+  "pumps": {
+    "mongo-pump-aggregate": {
+      "name": "mongo-pump-aggregate",
+      "meta": {
+        "mongo_url": "mongodb://username:password@{hostname:port},{hostname:port}/{db_name}",
+        "use_mixed_collection": true
```
Comment (komalsukhani) [inline tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: If `use_mixed_collection` is set to true, data is also stored in `tyk_analytics_aggregates` collection

File: tyk-docs/content/configure/tyk-pump-dashboard-config.md (hunk 1 above)
Comment (sedkis) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:45]: agreed

File: tyk-docs/content/configure/tyk-pump-dashboard-config.md (hunk 2 above)
Comment (sedkis) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: what if it is set to false? @komalsukhani

**Review by sedkis — COMMENTED** [review]
added the aggregate hint you mentioned

File: tyk-docs/content/configure/tyk-pump-dashboard-config.md (hunk 2 above)
Comment (sedkis) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: i've just tested it, the aggregate data in `z_tyk_analytics_aggregates` gets updated regardless of the value of `use_mixed_collection`

Comment (sedkis) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: Here's the code that runs when `used_mixed_collection` is enabled, I have no clue what it's doing

Comment (sedkis) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: https://github.com/TykTechnologies/tyk-pump/blob/master/pumps/mongo_aggregate.go#L302

Comment (komalsukhani) [reply tyk-docs/content/configure/tyk-pump-dashboard-config.md:61]: You are right. `z_tyk_analytics_aggregates` collection is updated no matter what is value of `use_mixed_collection`. If `use_mixed_collection` is set to true, data is also stored in `tyk_analytics_aggregates` collection

**Comment by sedkis** [comment]
@marksou , I'll hand this over to you.  we're good to go

**Pull request merged**